pkg/console/console.go

```diff
@@ -58,11 +65,41 @@ func Run() error {
 	restful.Filter(cors.Filter)
 	restful.Filter(restful.OPTIONSFilter())
 
+	tlsConfigWatch := tlsconfig.NewWatch(
+		path.Join(configDir, TlsProfileFile),
+		serviceCertPath,
+		serviceKeyPath,
+	)
+	tlsConfigWatch.Reload()
+
+	watchDone := make(chan struct{})
+	defer close(watchDone)
+	go func() {
+		err := tlsConfigWatch.Run(watchDone)
+		log.Log.Errorf("Error running TLS config watch: %s", err)
+	}()
+
 	server := &http.Server{
 		Addr: fmt.Sprintf("%s:%d", defaultAddress, defaultPort),
```

Comment:
Shouldn't we get that value from environment variable? (e.g. `SERVER_ADDR=0.0.0.0:8768`). Currently `defaultPort` it's a const variable.

Reply:
These address and port are only used inside the container.
We could make them configurable in a future PR, if it will be useful.